andela/kenya-cfh
# feature(landing page design):landing page

#### What does this PR do?
- This PR allows a user to see the landing page on visiting the site
- Helps a user get a brief info about the game

Description of Task to be completed?
N/A

#### What should this be manually tested?
- Run npm start to start the server
- Go to localhost:3000 to see the landing page

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
- Users should see a landing page with effective UI/UX #153756405

#### Screenshots (if appropriate)
<img width="1439" alt="screen shot 2018-01-15 at 9 41 09 pm" src="https://user-images.githubusercontent.com/28443813/34960996-8ac66d16-fa3d-11e7-8f06-76c68dcaab38.png">

#### Questions:
N/A
